hack/terraform-quickstart/genlogs.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export IDENT=${IDENT:-${HOME}/.ssh/id_rsa}
+export WORKER_IPS=`terraform output -json worker_ips | jq -r '.value[]'`
+export MASTER_IPS=`terraform output -json master_ips | jq -r '.value[]'`
+export BOOTSTRAP_IP=`terraform output bootstrap_node_ip`
+export LOGS_DIR=${LOGS_DIR:-/tmp/logs}
+export REMOTE_PORT=${REMOTE_PORT:-22}
```

Comment:
I'd like to avoid options that don't ever take more than one value. Can we remove a few of these?

Reply:
`more than one value`... those are default values if the variable does not have an overriding environment variable